site/layouts/shortcodes/example.html

```diff
@@ -4,12 +4,14 @@
   `args` are all optional and can be one of the following:
     * id: the `div`'s id - default: ""
     * class: any extra class(es) to be added to the `div` - default: ""
+    * javascript: extra JavaScript file to include
```

Comment:
`javascript` is maybe not precise enough; could be `js_filename` or `js_file` :shrug:

Reply:
js_file is fine I suppose :)

Reply:
Resolved via https://github.com/twbs/bootstrap/pull/36127/commits/c39bfc8d5d82fc3f5d44a4ea1009d5f15809737d